kaggle_environments/utils.py

```diff
@@ -113,12 +114,16 @@ def get_exec(raw, fallback=None):
         env = {}
         exec(code_object, env)
         sys.stdout = orig_out
-        print(buffer.getvalue())
+        output = buffer.getvalue()
+        if output:
+            print(output)
         return env
     except Exception as e:
         sys.stdout = orig_out
-        print(buffer.getvalue())
-        if fallback != None:
+        output = buffer.getvalue()
+        if output:
```

Comment:
This change gets rid of most of those annoying blank lines in the output

Reply:
👏